### PR TITLE
Various minor fixes

### PR DESCRIPTION
**Pull request by gtamazian**
The fixes were motivated by the **clang** C++ compiler warnings.

### CODE REVIEW

File: src/split/splitBed.cpp [hunk 1]
```diff
@@ -148,7 +148,7 @@ std::FILE* BedSplit::saveFileChunk(std::string& filename,size_t file_index)
     
     char tmp[10];
     filename.assign(this->outfileprefix);
-    sprintf(tmp,"%05d",(file_index+1));
+    sprintf(tmp,"%05lu",(file_index+1));
```
Comment (jmarshall) [inline src/split/splitBed.cpp]: `%05zu`; but probably it would be best to just avoid the issue by building the filename via an `ostringstream`.

**Comment by jmarshall** [comment]
PR #233 has a better fix for the overflow warning that makes `int2str()` work correctly for large numbers.

**Comment by arq5x** [comment]
Thanks all. We will try to merge in all of these patches soon.

File: src/split/splitBed.cpp (hunk 1 above)
Comment (gtamazian) [inline src/split/splitBed.cpp]: Thanks, I fixed it in f9001c5.

**Comment by gtamazian** [comment]
Fixed some tests; see also PR #239 for fixes of **bedtools coverage** and **bedtools sample** tests. Also possible overflows were fixed for **bedtools slop** in 5e017b63155f8a7f3e43a2e551bcfc535cd5bf0a.

**Comment by gtamazian** [comment]
Changes from this pull request were split to several branches for more convenient review; please check PR #249, #250 and #251.

**Pull request closed without merging**